packages/webpack/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@kubev2v/webpack",
+  "version": "0.0.1",
+  "description": "Allows building dynamic plugin assets with webpack",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kubev2v/forklift-console-plugin.git",
+    "directory": "packages/lib-webpack"
+  },
+  "files": ["./dist/*"],
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "clean": "rm -rf dist",
+    "clean:all": "npm run clean && rm -rf node_modules",
+    "build": "rm -rf ./dist && npx tsc -p ./tsconfig.json"
+  },
+  "devDependencies": {
+    "@openshift-console/dynamic-plugin-sdk": "0.0.18",
```

Comment:
1. the plugin uses different version `0.0.17` - is this on purpose?
2. after running the build the webpack folder has `node_modules`  folder with 2 packages - `@openshift-console`  and `@patternfly`

Reply:
thanks, :fire: fixed